psd/en/tutorial/add-text-effects/_index.md

```diff
@@ -0,0 +1,32 @@
+---
+title: Aspose.PSD Add Text Effects
+weight: 7700
+limit: 
+description: Learn hot to edit and export PSD file by code.
+keywords: [text effects, text shadow, photoshop text effects, text stroke, open photoshop file, psd file export, text effect psd]
+url: tutorial/add-text-effects/
+---
+
+{{< blocks/products/pf/main-wrap-class >}}
+{{< blocks/products/pf/main-container >}}
+{{< blocks/products/pf/feature-page-section h2="Learn How To Add Effects To TextLayer" >}}
+
+
+<p>
+In this tutorial, we'll create a text layer in a PSD file and apply different effects to it, such as shadows and strokes. Text layers are a powerful feature of the PSD format, and adding effects to them can make your designs more visually appealing and attractive.
+</p>
+
+<p>
+We'll start by creating a new text layer using the Aspose.PSD library and customizing its font, size, and color. Then, we'll add a shadow effect to the text layer to give it depth and dimension. Finally, we'll add a stroke effect to the text layer to create a border around the text.
```

Comment:
please add one more URL for keyword "text effect" with the link to our resources - apps, api reference, docs or etc. page should have the similar theme to this page